### PR TITLE
Javascript/javascript2/week3

### PR DESCRIPTION
**Pull request by Gohar33**

### CODE REVIEW

File: javascript/javascript2/week3/fastestPresserGame.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+let numberS = 0;
+let numberL = 0;
+
+
+function winnerFunction() {
+    if (numberS > numberL) {
+        return resultS.innerHTML = "Winner is a S player"
+    } else if (numberL > numberS) {
+        return resultL.innerHTML = "Winner is a L player"
+    }
```
Comment (cameronjdavis) [inline javascript/javascript2/week3/fastestPresserGame.js:17]: What about a draw?
```suggestion
    } else {
        // TODO
    }
```

File: javascript/javascript2/week3/fastestPresserGame.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    setTimeout(function () {
+        gameRunning = false;
+        winnerFunction()
+    }, contentNumber * 1000)
+})
+
+
+
+document.addEventListener("keypress", (event) => {
+    if (gameRunning === true) {
```
Comment (cameronjdavis) [inline javascript/javascript2/week3/fastestPresserGame.js:35]: `gameRunning === true` is the same as `gameRunning`
```suggestion
    if (gameRunning) {
```

**Review by cameronjdavis — CHANGES_REQUESTED** [review]
Well done on the Google Map.

File: javascript/javascript2/week3/fastestPresserGame.js (hunk 2 above)
Comment (Gohar33) [reply javascript/javascript2/week3/fastestPresserGame.js:35]: Thank you Cameron, I'll implement this feedback right away :)

**Pull request merged**